openweathermap.py

```diff
@@ -13,109 +16,160 @@ def __init__(self, **kwargs):
+                    "temp_today_temp_min": temp_today_temp_min,
+                    "pressure_today_press": pressure_today_press,
+                    "pressure_today_sea_level": pressure_today_sea_level,
+                    "humidity_today": humidity_today,
+                    "wind_today_deg": wind_today_deg,
+                    "wind_today_speed": wind_today_speed,
+                    "snow_today": snow_today,
+                    "rain_today": rain_today,
+                    "clouds_coverage_today": clouds_coverage_today,
+
+                    "weather_tomorrow": weather_tomorrow_status,
+                    "sunset_time_tomorrow": sunset_time_tomorrow,
+                    "sunrise_time_tomorrow": sunrise_time_tomorrow,
+                    "temp_tomorrow_temp": temp_tomorrow_temp,
+                    "temp_tomorrow_temp_max": temp_tomorrow_temp_max,
+                    "temp_tomorrow_temp_min": temp_tomorrow_temp_min,
+                    "pressure_tomorrow_press": pressure_tomorrow_press,
+                    "pressure_tomorrow_sea_level": pressure_tomorrow_sea_level,
+                    "humidity_tomorrow": humidity_tomorrow,
+                    # "wind_tomorrow_deg": wind_tomorrow_deg,
+                    "wind_tomorrow_speed": wind_tomorrow_speed,
+                    "snow_tomorrow": snow_tomorrow,
+                    "rain_tomorrow": rain_tomorrow,
+                    "clouds_coverage_tomorrow": clouds_coverage_tomorrow
+                }
+
+                self.say(message)
+            # Daily forecast
+            if self.day:
+                forecast = self.GetForecast(extended_location)
+                self.say(forecast)
+                    
+    def GetForecast(self, location):
+        # We dont use pyown for the daily forecast, and we don't won't to break the current config so we convert the units to the required parameter for the API call
+        if self.temp_unit == 'celsius':
+            self.temp_unit = 'metric'
+        if self.temp_unit == 'fahrenheit':
+            self.temp_unit = 'imperial'
+        if self.temp_unit == 'kelvin':
+            self.temp_unit = 'kelvin'
+ 
+        # We use the API for the 5 day / 3 hours --> https://openweathermap.org/forecast5
+        # Don't know why but we get a return of 7 days, so we can ask the weather for the next 7 days.
+        url = "http://api.openweathermap.org/data/2.5/forecast/daily?q=" + location + "&lang=" + self.lang + "&APPID=" + self.api_key + "&units=" + self.temp_unit
+        response = requests.get(url)
+        data = response.json()
+        
+        # To get the forecast for today we dont need self.days, so we update the dictionary with the current day
+        if self.day == 'today':
+            self.days.update({datetime.now().strftime('%A') : 'today'})
+
+        forecast = {}
+        forecast.update({'forecast_city' : data['city']['name']})
+        for f in data['list']:   
```

Comment:
don't use only a letter for your variables. Use comple words so we do know with what we are playing when reading the code.